app/controllers/application_controller.rb

```diff
@@ -67,7 +67,7 @@ def render_error_response(exception)
   def check_subscription
     # This block is left over from the initial version of chatwoot
     # We might reuse this later in the hosted version of chatwoot.
-    return unless ENV['BILLING_ENABLED']
+    return if !ENV['BILLING_ENABLED'] || !current_user
```

Comment:
let's get rid of the `!ENV['BILLING_ENABLED'] `check now that it's handled in the before action